components/Display.tsx

```diff
@@ -20,161 +20,189 @@ import InfoMarquee from "./Marquee";
         />
     */
-    return (
-        <div id="main" ref={mainRef} className="relative top-1/2 left-1/2 overflow-hidden w-main h-main">
-            <img className="block max-h-full max-w-full" src="/images/template-4k.png" alt="background" />
-            <SlideBg />
-            {isReady && <SlidesContainer setMainVol={setMainVol} />}
-            {locInfo.timezone !== "" && <DateTime tz={locInfo.timezone} />}
-            {locInfo.city !== "" && <div
-                id="city"
-                className="font-interstate font-semibold text-city pt-city-t absolute text-left ml-city-l w-city h-city top-city-t left-0 leading-city flex items-center transform scale-x-103 scale-y-100 origin-left"
-            >{locInfo.city}</div>}
-            <CCIcon iconCode={currentInfo.icon} windData={currentInfo.windSpeed} />
-            <Current
-                key={currentInfo.phrase}
-                temp={currentInfo.temp}
-                info={currentInfo}
-            />
-            <LogoArea />
-            <InfoMarquee
-                top="Hello World"
-                bottom="We have currently partnered with Indigo Wireless to offer great wireless service in Tioga County! Go to indigowireless.com or stop in at 100 Main in Wellsboro for more information on this promo. | Save $5.00 a month with easy, painless auto pay system. Sign up Today! | In weeks to come we will be making upgrades to our network to serve your TV experience better! You may experience brief No Signal mesages on your TV. If the message stays on your TV for more than 4 hours please reboot the TV and call us. | Remember that this is all made possible with help from the Weather Ranch Discord Server! | Help from MapGuy11, Goldblaze, and TWCJon! | To stay up to date with all the latest on the emulator join the Discord Server! https://discord.gg/4TpAsRtsAx | NextJs version inspired from https://github.com/buffbears/Weatherscan |"
-            />
-            {focusedAlert && <MarqueeSevere
-                top={focusedAlert.eventDescription}
-                bottom={focusedAlertText}
-            />}
+  return (
+    <div
+      id="main"
+      ref={mainRef}
+      className="relative top-1/2 left-1/2 overflow-hidden w-main h-main"
+    >
+      <img
+        className="block max-h-full max-w-full"
+        src="/images/template-4k.png"
+        alt="background"
+      />
+      <SlideBg />
+      {isReady && <SlidesContainer setMainVol={setMainVol} />}
+      {locInfo.timezone !== "" && <DateTime tz={locInfo.timezone} />}
+      {locInfo.city !== "" && (
+        <div
+          id="city"
+          className="font-interstate font-semibold text-city pt-city-t absolute text-left ml-city-l w-city h-city top-city-t left-0 leading-city flex items-center transform scale-x-103 scale-y-100 origin-left"
+        >
+          {locInfo.city}
         </div>
-    );
+      )}
+      <CCIcon iconCode={currentInfo.icon} windData={currentInfo.windSpeed} />
```

Comment:
Current conditions icon does not properly load. See https://openweathermap.org/weather-conditions for more info on implementing.